mlfinlab/ensemble/sb_bagging.py

```diff
@@ -0,0 +1,660 @@
+"""
+Implementation of Sequentially Bootstrapped Bagging Classifier using sklearn's library as base class
+"""
+import numbers
+import itertools
+from warnings import warn
+from abc import ABCMeta, abstractmethod
+import pandas as pd
+import numpy as np
+
+from sklearn.tree import DecisionTreeClassifier, DecisionTreeRegressor
+from sklearn.metrics import accuracy_score, r2_score
+from sklearn.ensemble.bagging import BaseBagging, BaggingClassifier, BaggingRegressor
+from sklearn.base import ClassifierMixin, RegressorMixin
+from sklearn.ensemble.base import _partition_estimators
+from sklearn.utils.random import sample_without_replacement
+from sklearn.utils.validation import has_fit_parameter
+from sklearn.utils import check_random_state, indices_to_mask, check_array, check_consistent_length, check_X_y
+from sklearn.utils._joblib import Parallel, delayed
+
+from mlfinlab.sampling.bootstrapping import seq_bootstrap, get_ind_matrix
+
+MAX_INT = np.iinfo(np.int32).max
+
+
+# pylint: disable=too-many-ancestors
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=too-many-branches
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-statements
+# pylint: disable=invalid-name
+# pylint: disable=protected-access
+# pylint: disable=len-as-condition
+# pylint: disable=attribute-defined-outside-init
+# pylint: disable=bad-super-call
+# pylint: disable=no-else-raise
+
+
```

Comment:
Some argument descriptions would be great here, even if these functions are not intended to be user-facing.